docs/user/sampling.md

```diff
@@ -0,0 +1,74 @@
+# Sampling
+
+`LocalRunner` gets trajectories through Sampling to train the policy. In
+Garage, [`LocalRunner`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/experiment/local_runner.py)
+uses [`Sampler`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/sampler/sampler.py)
+to perform sampling. `Sampler` manages [`Worker`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/sampler/worker.py)s
+and assign specific tasks to them, which is doing rollouts with agents and
+environments. You can also implement your own sampler and worker. The
+followings introduce the existing samplers and workers in Garage.
+
+## Sampler
+
+Sampler is responsible for assign sampling jobs to workers. Garage now has
+two types of `Sampler`s:
+
+- [`LocalSampler`](https://github.com/rlworkgroup/garage/blob/a767dcb95988abc14165a2a320ee39932a1c85ca/src/garage/sampler/local_sampler.py),
+the default sampler, which runs workers in the main process in serial style.
+With this sampler, all the sampling tasks will run in the same thread.
+
+- [`RaySampler`](https://github.com/rlworkgroup/garage/blob/master/src/garage/sampler/ray_sampler.py),
```

Comment:
What about MultiprocessingSampler?

Reply:
KR said that he will add it when it works with more backends.